uwreact_bringup/launch/zeds.launch

```diff
@@ -0,0 +1,71 @@
+    <arg name="publish_map_tf"                default="true" />
+    <arg name="base_frame"                    default="zed_camera_center" />
+
+    <arg name="depth_topic"                   default="depth_registered" />
+    <arg name="depth_cam_info_topic"          default="depth/camera_info" />
+    <arg name="laserscan_topic"               default="laserscan"/>
+
+    <!-- First ZED camera -->
+    <group ns="zed1">
+        <node name="zed_wrapper" pkg="zed_wrapper" type="zed_wrapper_node" output="screen" required="true">
+            <param name="zed_id"                value="$(arg zed_id1)" />
+            <param name="camera_model"          value="0" />
+            <param name="verbose"               value="$(arg verbose)" />
+            
+            <param name="resolution"            value="$(arg resolution)" />
+            <param name="frame_rate"            value="$(arg frame_rate)" />
+            <param name="gain"                  value="100" />
+            <param name="exposure"              value="100" />
+            <param name="auto_exposure"         value="true" />
+            <param name="mat_resize_factor"     value="1.0" />
+            
+            <param name="quality"               value="1" />
+            <param name="sensing_mode"          value="0" />
+            <param name="openni_depth_mode"     value="0" />
+            <param name="confidence"            value="100" />
+            <param name="max_depth"             value="10.0" />
+            <param name="depth_stabilization"   value="1" />
+            
+            <param name="pose_smoothing"        value="false" />
+            <param name="spatial_memory"        value="false" />
+            <param name="floor_alignment"       value="false" />
+            <param name="camera_flip"           value="false" />
+            <param name="publish_tf"            value="$(arg publish_tf)" />
+            <param name="publish_map_tf"        value="$(arg publish_map_tf)" />
+            <param name="init_odom_with_first_valid_pose"   value="true" />
+            <param name="publish_pose_covariance"   value="false" />
+
+            <param name="depth_topic"           value="$(arg depth_topic)" />
+            <param name="depth_cam_info_topic"  value="$(arg depth_cam_info_topic)" />
+            <param name="base_frame"            value="$(arg base_frame)" />
+        </node>
+
+        <node name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan">
+            <remap from="image"            to="$(arg depth_topic)" />
+            <remap from="camera_info"      to="$(arg depth_cam_info_topic)" />
+            <remap from="scan"             to="$(arg laserscan_topic)"/>
+
+            <param name="output_frame_id"  value="$(arg base_frame)" />
+            <!-- TODO(wraftus): Play around with scan_height and range_min -->
+            <param name="scan_height"      value="10"/> 
```

Comment:
scan_height and range_min should be args we can cahnge at runtime

Reply:
Resolved in 30accfd